template/.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Netlify Release
```

Comment:
Should we rename this file to `deploy.yml` to make it consistent with other projects?
```suggestion
name: Deploy to Netlify
```

Reply:
1d366c0 ✅